src/Hosting/WindowsServices/src/WindowsServiceLifetime.cs

```diff
@@ -14,9 +14,9 @@ public class WindowsServiceLifetime : ServiceBase, IHostLifetime
     {
         private readonly TaskCompletionSource<object> _delayStart = new TaskCompletionSource<object>(TaskCreationOptions.RunContinuationsAsynchronously);
         private readonly ManualResetEventSlim _delayStop = new ManualResetEventSlim();
-        private readonly HostOptions _hostOptions;
+        private readonly WindowsServiceLifetimeOptions _options;
 
-        public WindowsServiceLifetime(IHostEnvironment environment, IHostApplicationLifetime applicationLifetime, ILoggerFactory loggerFactory, IOptions<HostOptions> optionsAccessor)
+        public WindowsServiceLifetime(IHostEnvironment environment, IHostApplicationLifetime applicationLifetime, ILoggerFactory loggerFactory, IOptions<WindowsServiceLifetimeOptions> optionsAccessor)
```

Comment:
This requires a ref assembly update. See https://github.com/aspnet/Extensions/blob/master/docs/ReferenceAssemblies.md